python/common/python/platform.java

```diff
@@ -27,6 +27,8 @@
             platform_class_name = "python._platform.AndroidPlatform";
         } else if (vendor.equals("Azul Systems, Inc.")) {
             platform_class_name = "python._platform.JavaPlatform";
+        } else if (vendor.equals("JetBrains s.r.o")) {
+            platform_class_name = "python._platform.JavaPlatform";
         } else {
             throw new org.python.exceptions.RuntimeError("Unknown platform vendor '" + vendor + "'");
```

Comment:
To prevent this problem from happening in the future, I believe it will be good to stop raising this exception and just do this:

```
if (vendor.equals("The Android Project")) {
    platform_class_name = "python._platform.AndroidPlatform";
} else {
    platform_class_name = "python._platform.JavaPlatform";
}
```

Do you mind doing that change?

Reply:
Here we go

Reply:
That's a good suggestion.